.github/scripts/generate_ci_workflows.py

```diff
@@ -116,6 +116,16 @@ def __post_init__(self) -> None:
             self.only_build_on_pull_request = False
             self.ciflow_config.enabled = False
             self.ciflow_config.reset_root_job()
+
+        # The following code allows for scheduled jobs to be debuggable by
+        # adding the label 'ciflow/scheduled' and assigning + unassigning pytorchbot.
+        if self.is_scheduled:
+            self.on_pull_request = True
+            self.ciflow_config = CIFlowConfig(
```

Comment:
should we check if ciflow_config is already set?

I think it's better set by the individual workflows instead of doing it for all scheduled jobs, because the individual workflows can have their own set of labels to react, instead of just `ciflow/scheduled`

Reply:
Ah, fair. I will guard this with if self.ciflow_config.enabled is False.